core/deployment/src/main/java/io/quarkus/deployment/util/ExecUtil.java

```diff
@@ -0,0 +1,62 @@
+
+package io.quarkus.deployment.util;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+public class ExecUtil {
+
+    /**
+     * Execute the specified command from within the current directory.
+     * 
+     * @param command The command
+     * @param args The command arguments
+     * @return true if commands where executed successfully
+     */
+    public static boolean exec(String command, String... args) {
+        return exec(new File("."), command, args);
+    }
+
+    /**
+     * Execute the specified command from within the specified directory.
+     * 
+     * @param directory The directory
+     * @param command The command
+     * @param args The command arguments
+     * @return true if commands where executed successfully
+     */
+    public static boolean exec(File directory, String command, String... args) {
```

Comment:
I wonder if it would be better to have the command executed on another thread, like is done at :https://github.com/quarkusio/quarkus/blob/master/core/deployment/src/main/java/io/quarkus/deployment/pkg/steps/NativeImageBuildStep.java#L278

Reply:
I am not sure why they are pumping the output stream in a different thread.
I wouldn't mind do that same here, but I'd like to know what's the benefit.

Reply:
It is done that way because we were filtering the error stream and enhancing error messages. If you are dealing with two streams you need multiple threads.

Reply:
Thanks for the explanation

Reply:
This has been fixed.